app/src/main/java/com/mk/tasky/authentication/login/data/remote/LoginApi.kt

```diff
@@ -0,0 +1,20 @@
+package com.mk.tasky.authentication.login.data.remote
+
+import com.mk.tasky.BuildConfig
+import com.mk.tasky.authentication.login.data.remote.dto.LoginBodyDto
+import com.mk.tasky.authentication.login.data.remote.dto.LoginResponseDto
+import retrofit2.http.Body
+import retrofit2.http.Header
+import retrofit2.http.POST
+
+interface LoginApi {
+    companion object {
+        const val BASE_URL = "https://tasky.pl-coding.com/"
+    }
+
+    @POST("/login")
+    suspend fun login(
+        @Body body: LoginBodyDto,
+        @Header("x-api-key") apiKey: String = BuildConfig.API_KEY
```

Comment:
If a header is required for all routes, it might be more convenient to write a little interceptor for this. Just as a thought, there's no general right or wrong here :)

Reply:
Awesome! I'll apply this